keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2689,90 +2747,74 @@ previous answers based on experience in the field.
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-<!--
-Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-checking if there are objects with field X set) may be a last resort. Avoid
-logs or events for this purpose.
--->
+There will be pods which have a non-empty PodSpec.ResourceClaims field and ResourceClaim objects.
 
-Metrics in kube-scheduler (names to be decided):
-- number of classes using structured parameters
+Metrics in kube-controller-manager (names to be decided):
```

Comment:
Are we still deciding on the names?

Reply:
Yes. I'll propose something after I have looked at the code - probably in a POC PR.

Reply:
https://github.com/kubernetes/kubernetes/pull/127661

Let's review the metric names in that PR, then I'll update the KEP.

Reply:
The metric names look good to me.